src/Shared/LanguageSupport/IsExternalInit.cs

```diff
@@ -6,7 +6,6 @@
 using System.ComponentModel;
 
 namespace System.Runtime.CompilerServices;
-
 /// <summary>
```

Comment:
Nit: intentional line removal? Seems like a convention is to have a blank line after namespace.

Reply:
Weird! That was definitely unintended and I'm not sure how it happened. Will fix! Thank you!